### ISSUE TITLE
[BUG]: I don't think anything is working as intended.

### ISSUE THREAD

**Issue by Anantha-Kandrapu** [feder-cr/Jobs_Applier_AI_Agent#1014]
### Describe the bug

Run Main.py nothing happens , silent failures

### Steps to reproduce

run main.py

### Expected behavior

apply jobs

### Actual behavior

no action

### Branch

main

### Branch name

main

### Python version

3.11

### LLM Used

ChatGPT

### Model used

GPT-4o

**Comment by jericho1050** [comment]
#1011

Yeah, I think a lot has changed.
Before it used to apply for jobs, now it migrated to Resume Generator, which doesn't do anything.

**Comment by ddondada9** [comment]
apparently linkedin reached out to the developers and asked them to remove the linkedin code off the script

**Comment by JamesTsetsekas** [comment]
anyone know of a working project?

**Comment by erperejildo** [comment]
> apparently linkedin reached out to the developers and asked them to remove the linkedin code off the script

if that's true, there should be an old working version

**Comment by ddondada9** [comment]
Yes the older version works but they changed their code which made the code not run. Ask they did was change the xpath and I was able to figure out how to isolate the xpath but it’s nowhere  as advanced as this code. I only have one year of coding python. Sent from my iPhoneOn Dec 30, 2024, at 3:50 AM, Daniel ***@***.***> wrote:﻿

apparently linkedin reached out to the developers and asked them to remove the linkedin code off the script

if that's true, there should be an old working version

**Comment by steve2700** [comment]
bro wtf
